views/layouts/MainLayout/MainLayout.css

```diff
@@ -5,4 +5,5 @@
 .MainLayout_main {
   flex-shrink: 0;
   flex: 1;
+  margin-left: 253px;
```

Comment:
why the large margin-left..?

Reply:
we need the left margin to be equal to the width of the sidebar so that the sidebar doesn't cover up the content.